### DISCUSSION TITLE
Save vaporview settings (easily)

### DISCUSSION THREAD

**Discussion by louiecaulfield** [Lramseyer/vaporview#28] [Ideas]
I suggest a custom file extension (`.vaporview`?) for vaporview settings, which is the file you actually have opened when viewing a waveform.

This way, it makes sense to bind `CTRL+S` to `save vaporview settings`, and you can open the settings back by clicking on them in the explorer side bar.

**Comment by Lramseyer** [comment]
I was actually thinking about this, and associating that filetype with the extension. I'm going to take a look at what the Surfer and GTKwave folks are doing for saving and loading profiles, and see if I can support those formats. This way, if you decide you want to use a different application, or have coworkers who use a different application, things should be more interoperable.
